ui/css/itcss/components/newui-sections.scss

```diff
@@ -96,78 +87,6 @@
   }
 }
 
-// Side panel specific styles - all width constraints in one place
-.app--sidepanel {
-  // Main containers
-  .main-container {
-    max-width: 490px;
```

Comment:
No need to specify these multiple times, home screen is using RootLayout with a container that constraints the width